Cargo.toml

```diff
@@ -22,6 +22,7 @@ impl-trait-for-tuples = "0.2"
 deepsize = "0.2.0"
 deepsize_derive = "0.1.2"
 textwrap = "0.15.0"
+fnv = "1.0"
```

Comment:
If we want a fast hasher then [`rustc_hash`](https://docs.rs/rustc-hash/latest/rustc_hash/) is faster than `fnv`